commerce/domain/pricing/cart/PricingBasicCalculator/PricingCalculatorSample.cls

```diff
@@ -55,7 +56,8 @@ public class PricingCalculatorSample extends CartExtension.PricingCartCalculator
             // This is an example of throwing special type of Exception (CartCalculateRuntimeException).
             // Throwing this exception causes the rollback of all previously applied changes to the cart (in scope of given request)
             // and may not always be the best choice.
-            throw new CartExtension.CartCalculateRuntimeException('An integration error occurred in COMPUTE_PRICES. Contact your admin.');
+            System.debug('Exception occurred, message:' + e.getMessage() + ', stacktrace: ' + e.getStackTraceString());
+            throw new CartExtension.CartCalculateRuntimeException('An integration error occurred in COMPUTE_PRICES. Contact your admin', e);
```

Comment:
Do we need to do this or just do `throw e;` (Like you have in another apex class in same PR) here?

Reply:
I think, it was an example of exception which can cause(or can be used to) rollback. So, we can let it be there.
Comments are clearly mentioning the behaviour.